Fabric.Authorization.Domain/Services/RoleManager.cs

```diff
@@ -0,0 +1,100 @@
+﻿namespace Fabric.Authorization.Domain.Services

```

Comment:
Since we have a `RoleService`, I would suggest naming this something like `EdwAdminSyncService` or something more specific so that people don't get confused on the intent of this class.